data/strings/brands_strings.txt

```diff
@@ -38,15 +38,17 @@
     en = Blue Bottle Coffee
```

Comment:
As far as I can see, translations were generated from OSM. I can't find script to regenerate this file in repo, probably it never was committed or this work was done manually. 

It would be better implement a simple script to regeneraye this file from OSM source id you have cycles. Or please synchronize OSM and this file manually in your PR. Otherwise all manuals changes in PR will be nulled in the future.

Reply:
I do hope that all brands can be inputed directly from NSI with translation for other language from wikidata, will be a much easire environment to do

Reply:
I checked some brands and they matched wikidata translation. I am OK because re-generation from Wikidata and/or OSM should procude the same or closer result.